app/views/layouts/application.html.erb

```diff
@@ -6,15 +6,28 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job adding your cdn assets into a partial! This helps reduce the amount of code in the application layout file :)